packages/zoe/src/zoeService/zoe.js

```diff
@@ -9,6 +9,7 @@ import { makePromiseKit } from '@agoric/promise-kit';
  * Zoe uses ERTP, the Electronic Rights Transfer Protocol
  */
 import '@agoric/ertp/exported';
+import { MathKind } from '@agoric/ertp/src/amountMath';
```

Comment:
If we are making breaking changes to ERTP, what do you think about exporting amountMath from @agoric/ertp directly? I believe this would mean creating an `index.js` file so that we can export issuers.js and amountMath.js.

Reply:
Yeah, seems reasonable.